app/src/org/commcare/activities/SyncUIHandling.java

```diff
@@ -63,23 +63,35 @@ public static void handleSyncResult(ConnectorWithResultCallback activity,
 
     public static void handleSyncUpdate(CommCareActivity activity,
                                         Integer... update) {
-        if (update[0] == DataPullTask.PROGRESS_STARTED) {
+        int progressCode = update[0];
+        if (progressCode == DataPullTask.PROGRESS_STARTED) {
             activity.updateProgress(Localization.get("sync.progress.purge"), DataPullTask.DATA_PULL_TASK_ID);
-        } else if (update[0] == DataPullTask.PROGRESS_CLEANED) {
+        } else if (progressCode == DataPullTask.PROGRESS_CLEANED) {
             activity.updateProgress(Localization.get("sync.progress.authing"), DataPullTask.DATA_PULL_TASK_ID);
-        } else if (update[0] == DataPullTask.PROGRESS_AUTHED) {
+            activity.updateProgressBarVisibility(false);
```

Comment:
the progress bar was not actually utilized in this phase or in the PROGRESS_DOWNLOADING phase, so I think it makes sense to not show it, instead of just showing it as empty for a while